docs/ref/class-based-views/base.txt

```diff
@@ -70,11 +71,17 @@ MRO is an acronym for Method Resolution Order.
         attributes.
 
         When the view is called during the request/response cycle, the
-        :class:`~django.http.HttpRequest` is assigned to the view's ``request``
-        attribute. Any positional and/or keyword arguments :ref:`captured from
-        the URL pattern <how-django-processes-a-request>` are assigned to the
-        ``args`` and ``kwargs`` attributes, respectively. Then :meth:`dispatch`
-        is called.
+        :meth:`init` method assigns the :class:`~django.http.HttpRequest` to
+        the view's ``request`` attribute, and any positional and/or keyword
+        arguments :ref:`captured from the URL pattern
+        <how-django-processes-a-request>` to the ``args`` and ``kwargs``
+        attributes, respectively. Then :meth:`dispatch` is called.
+
+    .. method:: init(request, *args, **kwargs)
+
+        Initializes view instance attributes: ``self.request``, ``self.args``
```

Comment:
a, b, and c (missing comma after 'b')

also missing `.. versionadded:: 2.2` annotation

Also you mentioned a use case in the release notes but not here.